tests/test_jobcollection.py

```diff
@@ -18,10 +22,86 @@ def test_jobcollection(jobcollection_mock):
     assert len(jobcollection_mock.jobs) == 1
 
 
+def test_jobcollection_multiple(jobcollection_multiple_mock):
+    assert len(jobcollection_multiple_mock.jobs) == 2
+
+
+def test_jobcollection_download_result(jobcollection_mock):
```

Comment:
Test names always say "_result" instead of "_results" like the function. Not strict but would be good to fix in case someone searches for the test.

Reply:
Fixed